File: workers/print-weasy/environment.py
import os

PRINT_SERVER = os.getenv('PRINT_SERVER', 'http://print:3003/')
SENTRY_WORKER_PRINT_WEASY_DSN = os.getenv('SENTRY_WORKER_PRINT_WEASY_DSN', '')
AMQP_HOST = os.getenv('AMQP_HOST', 'rabbitmq')
AMQP_PORT = os.getenv('AMQP_PORT', '5672')
AMQP_VHOST = os.getenv('AMQP_VHOST', '/')
AMQP_USER = os.getenv('AMQP_USER', 'guest')
AMQP_PASS = os.getenv('AMQP_PASS', 'guest')

File: workers/print-puppeteer/environment.js
module.exports = {
  PRINT_SERVER: process.env.PRINT_SERVER || 'http://print:3003',
  SHARED_COOKIE_DOMAIN: process.env.SHARED_COOKIE_DOMAIN || 'print',
  SENTRY_WORKER_PRINT_PUPPETEER_DSN: process.env.SENTRY_WORKER_PRINT_PUPPETEER_DSN || '',
  AMQP_HOST: process.env.AMQP_HOST || 'rabbitmq',
  AMQP_PORT: process.env.AMQP_PORT || '5672',
  AMQP_VHOST: process.env.AMQP_VHOST || '/',
  AMQP_USER: process.env.AMQP_USER || 'guest',
  AMQP_PASS: process.env.AMQP_PASS || 'guest',
};
